src/Components/navButtonComponent/navButtonStyle.css

```diff
@@ -1,4 +1,4 @@
-.text {
+.text:not([href]):not([tabindex]) {

```

Comment:
cual es la intencion de este selector?

Reply:
Andres como usamos el link de react-scroll para manejar el scroll de pa pagina incial se afectaron los estilos, la idea era usar la especificidad para lograr consevar los estilos y esto **:not([href]):not([tabindex])** es lo que traia por defecto, ahora noto que se ve bastate extraño y feo asi que voy a usar otro tipo de especificidad en este proximo commit, me avisas si es mejor. gracias por las observaciones